rpc.go

```diff
@@ -34,8 +34,22 @@ type RPCMsgIn struct {
 // RPCMsgOut is sent back to the webserver
 type RPCMsgOut struct {
 	WAFResponse    int32
-	RequestID      string      `json:",omitempty"` // Set if the server expects an UpdateRequest with this ID (UUID)
-	RequestHeaders [][2]string `json:",omitempty"` // Any additional information in the form of additional request headers
+	RequestID      string      `json:",omitempty"`                  // Set if the server expects an UpdateRequest with this ID (UUID)
+	RequestHeaders [][2]string `json:",omitempty"`                  // Any additional information in the form of additional request headers
+	RespActions    []Action    `json:",omitempty" msg:",omitempty"` // Add or Delete application response headers
+}
+
+const (
```

Comment:
is there a companion PR yet for the other side?  I'm interested in whether there is a single definition for AddHdr etc.